src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskReplicationMIControllerTest.java

```diff
@@ -290,16 +587,69 @@ public void should_delete_task_and_action_recorded_in_replicate_db() {
             .body("task_history_list.get(1).update_action", equalTo("Configure"))
             .body("task_history_list.get(2).update_action", equalTo("Claim"))
             .body("task_history_list.get(3).state", equalTo("TERMINATED"))
-            .body("task_history_list.get(2).assignee", notNullValue())
+            .body("task_history_list.get(3).assignee", notNullValue())
```

Comment:
Similar to the comment below. Can we check terminationReason on task_history?

Reply:
Fixed